parameter_files/COLIBRE.yml

```diff
@@ -119,6 +119,22 @@ ApertureProperties:
     NumberOfGasParticles: true
     NumberOfStarParticles: true
     SpinParameter: true
+    TotalInertiaTensor: true
+    GasInertiaTensor: true
+    DarkMatterInertiaTensor: true
+    StellarInertiaTensor: true
+    TotalInertiaTensorReduced: false
```

Comment:
Why are we not computing some of these inertia tensor quantities?

Reply:
We don't want to compute them for most snapshots, but they are going to be used for z=0 to compare the methods